doc/docs/reference/complete-state.md

```diff
@@ -28,7 +28,7 @@ desiredState:
           metadata:
             name: hello-pod
           spec:
-            restartPolicy: Never
+            Restart: Never
```

Comment:
This is the pod spec of a workload. We should leave it as it was:
```suggestion
            restartPolicy: Never
```

Reply:
Ups, search and replace mistake. I will revert.

Reply:
It is reverted now.